src/js/post-911-gib-status/actions/post-911-gib-status.js

```diff
@@ -0,0 +1,77 @@
+// TODO: remove this hard-coded response once we can fetch from a vets-api endpoint
+const chapter33EnrollmentResponse = {
+  chapter33EducationInfo: {
+    dateOfBirth: '1977-10-01T04:00:00.000+0000',
+    firstName: 'Jean',
+    lastName: 'Picard',
+    regionalProcessingOffice: 'Central Office Washington, DC',
+    vaFileNumber: '301010301',
+    enrollmentList: [
+      {
+        beginDate: '2012-11-01T04:00:00.000+0000',
+        endDate: '2012-12-01T05:00:00.000+0000',
+        facilityCode: '11902614',
```

Comment:
Not relevant to this PR, but comparing this response to the UI, we'll have to roll up the enrollment objects by facility code on the front end since we're keeping the vets-api piece as thin as possible. That seems fine with me given there won't be hundreds of enrollments (I assume?), but want to make sure we're all on board with this happening on the client.